README.md

```diff
@@ -1,5 +1,10 @@
 # js-stellar-wallets
 
+> **Warning**
```

Comment:
Can you use headers in the warning message so the font is bigger please?

Reply:
I removed all the contents of the main README and kept just the warning.